_posts/2021-03-16-hhvm-4.101.markdown

```diff
@@ -0,0 +1,30 @@
+---
+title: "HHVM 4.101"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.101 is released! This release marks the end of support for 4.95;
+HHVM 4.96&ndash;4.100 remain supported, as do the 4.56 and 4.80 LTS releases.
+
+# Highlights
+
+- 'Variable $x is undefined, or not always defined' errors will no longer
+  suggest non-denotable internal variable names, such as `$0splice0`
+
+# Breaking Changes
+
+- `switch($x)` is now checked for exhaustivity for enum classes, instead of just
+  for normal enums
```

Comment:
This is a typechecker change, right? Probably worth mentioning that + mention that switch statements of any type are always checked for exhaustivity at runtime.